packages/datadog-instrumentations/src/jest.js

```diff
@@ -102,6 +103,7 @@ function getWrappedEnvironment (BaseEnvironment) {
       }
       if (event.name === 'test_done') {
         const asyncResource = asyncResources.get(event.test)
+        this.global._ddtrace = undefined;
```

Comment:
I don't think this is needed. It's fine if `_ddtrace` is available all throughout. Also, `handleTestEvent` is called with `test_done` multiple times whereas only once with `setup`